mmpb/extension/attributes/vc_assignments_impl.py

```diff
@@ -7,13 +7,15 @@
 import numpy as np
 from elf.io import open_file
 from pybdv.util import get_key
+from pybdv.metadata import get_data_path
 from vigra.analysis import extractRegionFeatures
 from vigra.sampling import resize
 from vigra.filters import distanceTransform
 
 
-def add_path_if_needed(file_path, dir_path):
-    return file_path if os.path.exists(file_path) else os.path.join(dir_path, file_path)
+def get_n5(path):
```

Comment:
I am not sure if this behaviour is not too confusing, because it will potentially switch the path even if explicit n5/h5 files are given.
Maybe we could instead allow passing xmls and then get the path from the xml?

Reply:
Ok, this only changes the behaviour in `main`, so this is fine.